### ISSUE TITLE
Chrome or Firefox Extension

### ISSUE THREAD

**Issue by tarbib** [ether/etherpad-lite#3468]
Hello

I could not find any extensions, I was wondering if there were any plans to add a feature similar to this [one](https://chrome.google.com/webstore/detail/google-docs-quick-create/bldgenmjegcnjebiongilahhcjldgmlm). It's very convenient to create a new pad from the browser without entering etherpad server URL and clicking on the create button but directly create a new pad from the extension.

Basically having this button in my extension bar : 
![image](https://user-images.githubusercontent.com/2255433/44455688-3ce44000-a631-11e8-8993-2381e83cd254.png)

Thank you!

**Comment by anoymouserver** [comment]
You could use a bookmarklet to open a new pad on a certain Etherpad instance.

For example you could use [this generator](https://mrcoles.com/bookmarklet/).

<details>
<summary>Step-by-step instructions</summary>
<ul>
<li>paste the following code into the '<i>Enter your javascript code here:</i>' field</li>
<li>edit the last code line to match your Etherpad instance (replace <u>https://beta.etherpad.org</u>)
<li>enter a name for your bookmarklet at '<i>Name:</i>'</li>
<li>click the '<i>Convert to bookmarklet</i>' button</li>
<li>drag the light blue button to your bookmark bar or use the right-click menu to create a bookmark</li>
</ul>
<img src="https://user-images.githubusercontent.com/15173012/44718853-b791dd00-aac1-11e8-8373-d442185a9517.png"/>
</details>
<br>

```js
(function(e){
  var c = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
  var l = 10;
  var pad = '';
  for (var i = 0; i < l; i++) {
    var rnum = Math.floor(Math.random() * c.length);
    pad += c.substring(rnum, rnum + 1);
  }
  window.open(e + '/p/' + pad, '_blank');
})('https://beta.etherpad.org')
```

**Comment by tarbib** [comment]
@anoymouserver 
Yay! Thanks a lot

I just had to remove this "/" to make it work
![image](https://user-images.githubusercontent.com/2255433/44719583-56d0c780-aaf6-11e8-8faf-a0be375d9484.png)

perfect now

Thanks a lot !

**Comment by anoymouserver** [comment]
This `/` depends on whether your URL itself has a `/` at the end or not.

If this issue is solved for you, please close this ticket. :)

**Comment by tarbib** [comment]
Marvelous solution
